yarn/datadog_checks/yarn/yarn.py

```diff
@@ -231,19 +230,20 @@ def _yarn_app_metrics(self, rm_address, app_tags, addl_tags):
         """
         metrics_json = self._rest_request_to_json(rm_address, YARN_APPS_PATH, addl_tags)
 
-        if metrics_json and metrics_json['apps'] is not None and metrics_json['apps']['app'] is not None:
-            for app_json in metrics_json['apps']['app']:
-                tags = self._get_app_tags(app_json, app_tags) + addl_tags
+        if metrics_json and metrics_json.get('apps'):
+            if metrics_json['apps'].get('app') is not None:
```

Comment:
```suggestion
        if metrics_json and metrics_json.get('apps') and metrics_json['apps'].get('app') is not None:
```

This should work too, right?

Reply:
Yes, it'll simplify the logic. I'll update :)